packages/react/src/components/dialog/dialogActionButtons/DialogActionButtons.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { ReactNode, ReactNodeArray } from 'react';
+
+import styles from './DialogActionButtons.module.scss';
+
+export type DialogActionButtonProps = React.PropsWithChildren<{
```

Comment:
I wonder if the PropsWithChildren adds value here?

Reply:
No. I'll fix these.